ddtrace/contrib/flask/middleware.py

```diff
@@ -120,11 +121,19 @@ def _start_span(self):
         )
 
         try:
-            g.flask_datadog_span = self.app._tracer.trace(
+            g.flask_datadog_span = span = self.app._tracer.trace(
                 SPAN_NAME,
                 service=self.app._service,
                 span_type=SpanTypes.WEB,
             )
+
+            appsec.process_request(
```

Comment:
we don't need to modify this file at all, this middleware shouldn't be used by users anymore. ??

Reply:
if we do keep this, I wonder if we should have a separate `try/except` for `process_request`, we could do a better error message, or it might be a better pattern anyways since if it failing did cause tracing to stop working for this request, we want to avoid that if we can.

WDYT?